### ISSUE TITLE
Laravel 10 Compatibility

### ISSUE THREAD

**Issue by hossein** [dusterio/link-preview#38]
Thanks for your your work on the library. Please support Laravel 10.

**Comment by Umar-Farooq-Shafi** [comment]
@hossein do you find any alternative?

**Comment by DarkcoderSe** [comment]
have you?

**Comment by Umar-Farooq-Shafi** [comment]
no, I copied the code from the repo and changed it according to Laravel 10.

**Comment by ThePlatinum** [comment]
I built an alternative, it's much simpler too... Feel free to contribute:

https://github.com/Teners-net/laravel-link-preview
